CONTRIBUTING.md

```diff
@@ -78,21 +78,36 @@ See <https://docs.basedpyright.com/> for IDE integration and other info.
 
 ### mypy
 
-Mypy requires a bit more effort to run, since it doesn't play well with
-editable dependencies, and it doesn't seem to recognize NumType's `numpy-stubs`
-if `numpy` is also installed. You can get around that by running:
+Mypy is used for type-checking the codebase and the type-tests. It takes more effort to run it on the codebase than `basedpyright`.
+This is primarily a consequence of mypy's inconsequent prioritization of the stubs bundled
+with NumPy over the `numpy-stubs` within a local development environment.
+To get around this, you can delete all `*.pyi` from your **local** `numpy` installation directory by running `uv tool/unstub.py`.
+
+**General Type Checking**
+
+To run mypy generally (excluding the main numpy dependency group):
 
 ```shell
 uv run --no-editable --no-group=numpy mypy .
 ```
 
-In rare cases, the cache can get corrupted, which can lead to incorrect
-results. So in case the CI output of mypy differs from your local output, you
-should delete the `.mypy_cache` directory in the `numtype` root.
+**Cache Issues**
+
+In rare cases, the cache (`.mypy_cache` in the `numtype` root) can get corrupted,
+leading to incorrect results. If CI output differs from your local output, delete
+this directory.
 
 The documentation can be found at <https://mypy.readthedocs.io/>, and issues can
 be reported at <https://github.com/python/mypy>.
 
+### pytest
+
+Pytest runs runtime tests located in `src/*/@test`. It works out-of-the-box:
+
+```bash
+uv run pytest [OPTIONS]
+```
+
```

Comment:
Maybe a link to the pytest docs could help the reader in case they want to know what the 
`[OPTIONS]` are that they can use